py/server/tests/test_updateby.py

```diff
@@ -68,51 +67,81 @@ def test_simple_ops_proxy(self):
-                        ugp.auto_locking = False
+                            self.assertEqual(ct.size, rct.size)
 
     def test_ema_proxy(self):
         op_ctrl = OperationControl(on_null=BadDataBehavior.THROW,
                                    on_nan=BadDataBehavior.RESET,
                                    big_value_context=MathContext.UNLIMITED)
 
-        ema_ops = [ema_tick_decay(time_scale_ticks=100, cols="ema_a = a"),
-                          ema_tick_decay(time_scale_ticks=100, cols="ema_a = a", op_control=op_ctrl),
-                          ema_time_decay(ts_col="Timestamp", time_scale=10, cols="ema_a = a"),
-                          ema_time_decay(ts_col="Timestamp", time_scale=100, cols="ema_c = c",
-                                         op_control=op_ctrl),
-                          ]
+        ops = [ema_tick_decay(time_scale_ticks=100, cols="ema_a = a"),
+               ema_tick_decay(time_scale_ticks=100, cols="ema_a = a", op_control=op_ctrl),
+               ema_time_decay(ts_col="Timestamp", time_scale=10, cols="ema_a = a"),
+               ema_time_decay(ts_col="Timestamp", time_scale=100, cols="ema_c = c",
+                              op_control=op_ctrl),
+               ]
         pt_proxies = [self.static_table.partition_by("b").proxy(),
                       self.ticking_table.partition_by("b").proxy(),
                       ]
 
-        for ema_op in ema_ops:
-            with self.subTest(ema_op):
+        for op in ops:
+            with self.subTest(op):
                 for pt_proxy in pt_proxies:
-                    if pt_proxy.is_refreshing:
-                        ugp.auto_locking = True
-
-                    ema_proxy = pt_proxy.update_by(ema_op, by="e")
-                    for ct, ema_ct in zip(pt_proxy.target.constituent_tables, ema_proxy.target.constituent_tables):
-                        self.assertTrue(ema_ct.is_refreshing is ct.is_refreshing)
-                        self.assertEqual(len(ema_ct.columns), 1 + len(ct.columns))
+                    rt_proxy = pt_proxy.update_by(op, by="e")
+                    for ct, rct in zip(pt_proxy.target.constituent_tables, rt_proxy.target.constituent_tables):
+                        self.assertTrue(rct.is_refreshing is ct.is_refreshing)
+                        self.assertEqual(len(rct.columns), 1 + len(ct.columns))
                         with ugp.exclusive_lock():
-                            self.assertEqual(ct.size, ema_ct.size)
+                            self.assertEqual(ct.size, rct.size)
+
+    def test_rolling_sum(self):
+        ops = [
+            rolling_sum_tick(cols=["rsum_a = a", "rsum_d = d"], rev_ticks=10),
+            rolling_sum_tick(cols=["rsum_a = a", "rsum_d = d"], rev_ticks=10, fwd_ticks=10),
+            rolling_sum_time(ts_col="Timestamp", cols=["rsum_b = b", "rsum_e = e"], rev_time="00:00:10"),
+            rolling_sum_time(ts_col="Timestamp", cols=["rsum_b = b", "rsum_e = e"], rev_time="00:00:10",
```

Comment:
Can we add at least one test with `-hh:mm:ss` specifiers as in the documented examples for `rolling_sum_time()`?